includes/class-wc-payments.php

```diff
@@ -1039,6 +1039,7 @@ public static function ajax_init_platform_checkout() {
 				'store_api_url'     => self::get_store_api_url(),
 				'account_id'        => $account_id,
 				'test_mode'         => self::get_gateway()->is_in_test_mode(),
+				'capture_method'    => empty( self::get_gateway()->get_option( 'manual_capture' ) ) || 'no' === self::get_gateway()->get_option( 'manual_capture' ) ? 'automattic' : 'manual',
```

Comment:
Testing it with the WooPay PR when using the automatic method I am getting errors. This is because automattic is written with 2 T should be automatic